content/en/docs/tasks/administer-cluster/nodelocaldns.md

```diff
@@ -50,13 +50,15 @@ This is the path followed by DNS Queries after NodeLocal DNSCache is enabled:
 
 This feature can be enabled using the command:
 
-`KUBE_ENABLE_NODELOCAL_DNS=true go run hack/e2e.go -v --up`
+`KUBE_ENABLE_NODELOCAL_DNS=true kubetest --up`
```

Comment:
This change isn't making things worse so I think it's OK to accept; however, the page prerequisites don't set things up so that `kubetest` will work.

Reply:
running kubetest requires the user to setup the local clone of kubernetes repo and the kubernetes/test-infra repo. I am thinking this command is not suitable in the docs. I am working on a change to list the steps here, i will update this PR.